sycl/source/handler.cpp

```diff
@@ -202,15 +202,20 @@ event handler::finalize() {
         std::move(MRequirements), std::move(MEvents), MCGType, MCodeLoc));
     break;
   case detail::CG::NONE:
-    throw runtime_error("Command group submitted without a kernel or a "
-                        "explicit memory operation.",
-                        PI_INVALID_OPERATION);
+    if (detail::pi::trace(detail::pi::TraceLevel::PI_TRACE_ALL)) {
+      std::cout << "WARNING: An empty command group is submitted." << std::endl;
+    }
+    detail::EventImplPtr Event =
+        std::make_shared<cl::sycl::detail::event_impl>();
+    MLastEvent = detail::createSyclObjFromImpl<event>(Event);
+    return MLastEvent;
   }
 
-  if (!CommandGroup)
+  if (!CommandGroup) {
     throw sycl::runtime_error(
         "Internal Error. Command group cannot be constructed.",
         PI_INVALID_OPERATION);
+  }
```

Comment:
Nit: this particular change doesn't seem to be related to the change and AFAIK, the runtime library follows LLVM's coding style, which prefers no braces for one-statement if/else branches.
Anyway, I'll let @romanovvlad to decide on that change.

Reply:
@maximdimakov  Could you please remove the braces?

Reply:
Removed